dvc/lock.py

```diff
@@ -26,10 +27,56 @@ class LockError(DvcException):
     """Thrown when unable to acquire the lock for dvc repo."""
 
 
+class Lock(object):
+    """Class for dvc repo lock.
+
+    Uses zc.lockfile as backend.
+    """
+
+    def __init__(self, lockfile, friendly=False, **kwargs):
+        self._friendly = friendly
+        self.lockfile = lockfile
+        self._lock = None
+
+    @property
+    def files(self):
+        return [self.lockfile]
+
+    def _do_lock(self):
+        try:
+            with Tqdm(
```

Comment:
Kudos @shcheklein for suggesting this trick. 🙂

Reply:
A disappearing message is a smart trick, using `Tqdm` feels a bit dirty though)

Reply:
Dirty but ok I guess.